builtin/null/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+integration {
+  name        = "Null"
+  description = "TODO"
```

Comment:
@hashicorp/waypoint-ecosystem, Small callout here that there's still a TODO here